src/search/merge_and_shrink/merge_and_shrink_algorithm.h

```diff
@@ -32,12 +32,12 @@ class MergeAndShrinkAlgorithm {
 
     // Options for shrinking
     // Hard limit: the maximum size of a transition system at any point.
-    const int max_states;
+    int max_states;
     // Hard limit: the maximum size of a transition system before being merged.
-    const int max_states_before_merge;
+    int max_states_before_merge;
     /* A soft limit for triggering shrinking even if the hard limits
        max_states and max_states_before_merge are not violated. */
-    const int shrink_threshold_before_merge;
+    int shrink_threshold_before_merge;
 
     // Options for pruning
     const bool prune_unreachable_states;
```

Comment:
Since you are removing the `const`, I'd remove this one as well.

Reply:
Is it just a concern of consistency? Isn't it better to have values `const` if possible?

The value `shrink_threshold_before_merge` is changed inside the constructor and therefore non-const.
However, `prune_unreachable_states` is never changed and therefore const.